libs/ledgerHardwareWallet/ledgerLiskAppIPCChannel/serverLedgerHWCommunication/index.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable max-statements */
+import TransportNodeHid from '@ledgerhq/hw-transport-node-hid';
+import { LiskApp } from '@zondax/ledger-lisk';
+import { getDevicesFromPaths, getLedgerAccount } from './utils';
+
+export async function getPubKey({ devicePath, accountIndex }) {
+  let transport = TransportNodeHid;
+  try {
+    transport = await transport.open(devicePath);
+    const liskLedger = new LiskApp(transport);
+    const ledgerAccount = getLedgerAccount(accountIndex);
+    const account = await liskLedger.getAddressAndPubKey(ledgerAccount.derivePath());
+    await transport?.close();
+    const errorMessage = account?.error_message;
+    if (errorMessage === 'No errors') {
+      return account?.pubKey;
+    }
+    return Promise.reject(errorMessage);
+  } catch (error) {
+    await transport?.close();
+    return Promise.reject(error);
+  }
+}
+
+export async function getSignedTransaction({ devicePath, accountIndex, unsignedMessage }) {
+  let transport = TransportNodeHid;
+  try {
+    transport = await transport.open(devicePath);
```

Comment:
```suggestion
  let transport;
  try {
    transport = await TransportNodeHid.open(devicePath);
```

please fix this everywhere 🙏🏼

Reply:
fixed